src/clients/SpokePoolClient.ts

```diff
@@ -809,21 +809,7 @@ export class SpokePoolClient extends BaseAbstractClient {
     const tStart = Date.now();
     const query = await paginatedEventQuery(
       this.spokePool,
-      this.spokePool.filters.V3FundsDeposited(
-        null,
-        null,
-        null,
-        null,
-        destinationChainId,
-        depositId,
-        null,
-        null,
-        null,
-        depositor,
-        null,
-        null,
-        null
-      ),
+      this.spokePool.filters.V3FundsDeposited(null, null, null, null, null, depositId),
```

Comment:
I think this is fine:
- Same # of RPC requests are sent via paginatedEventQuery
- Since `depositId` is never duplicated in the SpokePool, the filter won't return too many events